src/main/java/org/jenkinsci/plugins/durabletask/FileMonitoringTask.java

```diff
@@ -179,9 +179,14 @@ String getCharset() {
         protected FileMonitoringController(FilePath ws) throws IOException, InterruptedException {
             // can't keep ws reference because Controller is expected to be serializable
             ws.mkdirs();
-            FilePath cd = tempDir(ws).child("durable-" + Util.getDigestOf(UUID.randomUUID().toString()).substring(0,8));
-            cd.mkdirs();
-            controlDir = cd.getRemote();
+            FilePath tmpDir = /* TODO pending JENKINS-61197 fix in baseline */ ws.getParent() != null ? WorkspaceList.tempDir(ws) : null;
```

Comment:
```suggestion
            FilePath tmpDir = ws.getParent() != null ? WorkspaceList.tempDir(ws) : null;
```
61197 has been resolved, so we can remove this comment. I don't think there needs to be anything else done since the null check is being done. Please correct me if I'm wrong.

Reply:
> 61197 has been resolved

But not in the `jenkins.version` currently in use.

> the null check is being done

Exactly my point—once the core fix _is_ in our baseline version, we can remove the null check and just have

```java
FilePath tmpDir = WorkspaceList.tempDir(ws);
```

Reply:
you're right, sorry, for some reason that part `in baseline` just went completely over my head. I'll go ahead and merge this in the next hour or so unless someone speaks up.